### PR TITLE
Converted improper epytext to proper epytext

### PR DESCRIPTION
**Pull request by acabhishek942**
See - https://twistedmatrix.com/trac/ticket/8497

### CODE REVIEW

File: twisted/conch/ssh/keys.py [hunk 1]
```diff
@@ -1233,19 +1233,19 @@ def objectType(obj):
 
 def _getPersistentRSAKey(location, keySize=4096):
     """
-    This function returns a persistent L{Key}.
+    This function returns a persistent C{Key}.
```
Comment (rodrigc) [inline twisted/conch/ssh/keys.py]: No, change back

File: twisted/conch/ssh/keys.py [hunk 2]
```diff
@@ -1233,19 +1233,19 @@ def objectType(obj):
...
 
     The key is loaded from a PEM file in C{location}. If it does not exist, a
     key with the key size of C{keySize} is generated and saved.
 
     @param location: Where the key is stored.
-    @type location: L{twisted.python.filepath.FilePath)
+    @type location: C{twisted.python.filepath.FilePath}
```
Comment (rodrigc) [inline twisted/conch/ssh/keys.py]: No, change back

File: twisted/conch/ssh/keys.py [hunk 3]
```diff
@@ -1233,19 +1233,19 @@ def objectType(obj):
...
 
     @param keySize: The size of the key, if it needs to be generated.
-    @type keySize: L{int}
+    @type keySize: C{int}
```
Comment (rodrigc) [inline twisted/conch/ssh/keys.py]: No, change back.

File: twisted/conch/ssh/keys.py [hunk 4]
```diff
@@ -1233,19 +1233,19 @@ def objectType(obj):
...
 
     @returns: A persistent key.
-    @rtype: L{Key}
+    @rtype: C{Key}
```
Comment (rodrigc) [inline twisted/conch/ssh/keys.py]: No, change back.

File: twisted/conch/ssh/transport.py [hunk 5]
```diff
@@ -107,8 +107,8 @@ class SSHCiphers:
     to encrypt and authenticate the SSH connection.
 
     @cvar cipherMap: A dictionary mapping SSH encryption names to 3-tuples of
-                     (<cryptography.hazmat.primitives.interfaces.CipherAlgorithm>,
-                      <block size>, <cryptography.hazmat.primitives.interfaces.Mode>)
+        (<cryptography.hazmat.primitives.interfaces.CipherAlgorithm>,
+        <block size>, <cryptography.hazmat.primitives.interfaces.Mode>)
```
Comment (rodrigc) [inline twisted/conch/ssh/transport.py:111]: OK

File: twisted/internet/interfaces.py [hunk 6]
```diff
@@ -1007,20 +1007,20 @@ def adoptDatagramPort(fileDescriptor, addressFamily, protocol,
             done as soon as C{adoptDatagramPort} returns.
         @type fileDescriptor: C{int}
 
-        @param addressFamily: The address family (or I{domain}) of the socket.
-            For example, L{socket.AF_INET6}.
+        @param addressFamily: The address family or I{domain} of the socket.
+            For example, C{socket.AF_INET6}.
```
Comment (rodrigc) [inline twisted/internet/interfaces.py]: No, change back.

File: twisted/internet/interfaces.py [hunk 7]
```diff
@@ -1007,20 +1007,20 @@ def adoptDatagramPort(fileDescriptor, addressFamily, protocol,
...
         @type addressFamily: C{int}
 
-        @param protocol: A L{DatagramProtocol} instance to connect to
+        @param protocol: A C{DatagramProtocol} instance to connect to
             a UDP transport.
-        @type protocol: L{DatagramProtocol}
+        @type protocol: C{DatagramProtocol}
```
Comment (rodrigc) [inline twisted/internet/interfaces.py]: No, change back

File: twisted/internet/interfaces.py [hunk 8]
```diff
@@ -1007,20 +1007,20 @@ def adoptDatagramPort(fileDescriptor, addressFamily, protocol,
...
 
         @param maxPacketSize: The maximum packet size to accept.
         @type maxPacketSize: C{int}
 
-        @return: An object providing L{IListeningPort}.
+        @return: An object providing C{IListeningPort}.
```
Comment (rodrigc) [inline twisted/internet/interfaces.py]: No, change back

File: twisted/internet/interfaces.py [hunk 9]
```diff
@@ -1007,20 +1007,20 @@ def adoptDatagramPort(fileDescriptor, addressFamily, protocol,
...
 
-        @raise L{UnsupportedAddressFamily}: If the given address family is not
+        @raise UnsupportedAddressFamily: If the given address family is not
```
Comment (rodrigc) [inline twisted/internet/interfaces.py:1023]: No, change back

File: twisted/mail/smtp.py [hunk 10]
```diff
@@ -1415,7 +1415,7 @@ def registerAuthenticator(self, auth):
         @param auth: The Authentication mechanism to register
         @type auth: L{IClientAuthentication} implementor
 
-        @return L{None}
+        @return: L{None}
```
Comment (rodrigc) [inline twisted/mail/smtp.py:1418]: OK

File: twisted/names/dns.py [hunk 11]
```diff
@@ -601,13 +601,13 @@ class _OPTHeader(tputil.FancyStrMixin, tputil.FancyEqMixin, object):
         attribute is a readonly property.
 
     @ivar type: The DNS record type. This is a fixed value of 41
-        (C{dns.OPT} for OPT Record. This attribute is a readonly
+        C{dns.OPT} for OPT Record. This attribute is a readonly
```
Comment (rodrigc) [inline twisted/names/dns.py:604]: OK

File: twisted/names/dns.py [hunk 12]
```diff
@@ -2455,12 +2455,12 @@ def __init__(self, id=0, answer=False, opCode=OP_QUERY, auth=False,
...
                  queries=None, answers=None, authority=None, additional=None):
         """
-        Construct a new L{_EDNSMessage}
+        Construct a new C{_EDNSMessage}
 
-        @see U{RFC1035 section-4.1.1<https://tools.ietf.org/html/rfc1035#section-4.1.1>}
-        @see U{RFC2535 section-6.1<https://tools.ietf.org/html/rfc2535#section-6.1>}
-        @see U{RFC3225 section-3<https://tools.ietf.org/html/rfc3225#section-3>}
-        @see U{RFC6891 section-6.1.3<https://tools.ietf.org/html/rfc6891#section-6.1.3>}
+        @see: U{RFC1035 section-4.1.1<https://tools.ietf.org/html/rfc1035#section-4.1.1>}
```
Comment (rodrigc) [inline twisted/names/dns.py:2460]: OK

File: twisted/names/dns.py [hunk 13]
```diff
@@ -2455,12 +2455,12 @@ def __init__(self, id=0, answer=False, opCode=OP_QUERY, auth=False,
...
+        @see: U{RFC2535 section-6.1<https://tools.ietf.org/html/rfc2535#section-6.1>}
```
Comment (rodrigc) [inline twisted/names/dns.py:2461]: OK

File: twisted/names/dns.py [hunk 14]
```diff
@@ -2455,12 +2455,12 @@ def __init__(self, id=0, answer=False, opCode=OP_QUERY, auth=False,
...
+        @see: U{RFC3225 section-3<https://tools.ietf.org/html/rfc3225#section-3>}
+        @see: U{RFC6891 section-6.1.3<https://tools.ietf.org/html/rfc6891#section-6.1.3>}
```
Comment (rodrigc) [inline twisted/names/dns.py:2463]: OK

File: twisted/names/dns.py [hunk 15]
```diff
@@ -2529,17 +2529,17 @@ def __init__(self, id=0, answer=False, opCode=OP_QUERY, auth=False,
             the requestor's network stack.
         @type maxSize: L{int}
 
-        @param queries: The L{list} of L{Query} associated with this message.
+        @param queries: The C{list} of C{Query} associated with this message.
```
Comment (rodrigc) [inline twisted/names/dns.py]: No, change back

File: twisted/names/dns.py [hunk 16]
```diff
@@ -2529,17 +2529,17 @@ def __init__(self, id=0, answer=False, opCode=OP_QUERY, auth=False,
...
         @type queries: L{list} of L{Query}
 
-        @param answers: The L{list} of answers associated with this message.
+        @param answers: The C{list} of answers associated with this message.
```
Comment (rodrigc) [inline twisted/names/dns.py]: No, change back

File: twisted/names/dns.py [hunk 17]
```diff
@@ -2529,17 +2529,17 @@ def __init__(self, id=0, answer=False, opCode=OP_QUERY, auth=False,
...
         @type answers: L{list} of L{RRHeader}
 
-        @param authority: The L{list} of authority records associated with this
+        @param authority: The C{list} of authority records associated with this
```
Comment (rodrigc) [inline twisted/names/dns.py]: No, change back

File: twisted/names/dns.py [hunk 18]
```diff
@@ -2529,17 +2529,17 @@ def __init__(self, id=0, answer=False, opCode=OP_QUERY, auth=False,
...
             message.
         @type authority: L{list} of L{RRHeader}
 
-        @param additional: The L{list} of additional records associated with
+        @param additional: The C{list} of additional records associated with
```
Comment (rodrigc) [inline twisted/names/dns.py]: No, change back.

File: twisted/names/server.py [hunk 19]
```diff
@@ -27,38 +27,38 @@
...
         C{clients}. Defaults to L{None} if no caches are specified. See
-        C{caches} of L{__init__} for more details.
-    @type cache: L{Cache<twisted.names.cache.Cache} or L{None}
+        C{caches} of C{__init__} for more details.
+    @type cache: C{Cache<twisted.names.cache.Cache} or C{None}
 
     @ivar canRecurse: A flag indicating whether this server is capable of
         performing recursive DNS resolution.
-    @type canRecurse: L{bool}
+    @type canRecurse: C{bool}
```
Comment (rodrigc) [inline twisted/names/server.py]: No, change back

File: twisted/python/logfile.py [hunk 20]
```diff
@@ -62,11 +62,12 @@ def _openFile(self):
         Open the log file.
 
         We don't open files in binary mode since:
-        * an encoding would have to be chosen and that would have to be
-          configurable
-        * Twisted doesn't actually support logging non-ASCII messages
-          (see U{https://twistedmatrix.com/trac/ticket/989})
-        * logging plain ASCII messages is fine with any non-binary mode.
+
```
Comment (rodrigc) [inline twisted/python/logfile.py:65]: Why is this change necessary?

File: twisted/trial/_dist/workerreporter.py [hunk 21]
```diff
@@ -21,7 +21,7 @@ class WorkerReporter(TestResult):
     Reporter for trial's distributed workers. We send things not through a
     stream, but through an C{AMP} protocol's C{callRemote} method.
 
-    :ivar str _DEFAULT_TODO: Default message for expected failures and
+    @ivar _DEFAULT_TODO: Default message for expected failures and
```
Comment (rodrigc) [inline twisted/trial/_dist/workerreporter.py:24]: OK, this change is good, it fixes a typo, **ivar str** should be **@ivar**

File: twisted/web/_http2.py [hunk 22]
```diff
@@ -1059,7 +1059,7 @@ def registerProducer(self, producer, streaming):
 
     def unregisterProducer(self):
         """
-        @see L{IConsumer.unregisterProducer}
+        @see: C{IConsumer.unregisterProducer}
```
Comment (rodrigc) [inline twisted/web/_http2.py]: No, The only part of this line which is ok is changing @see to @see:

File: twisted/web/_http2.py [hunk 23]
```diff
@@ -1073,22 +1073,22 @@ def unregisterProducer(self):
     # Implementation: IPushProducer
     def stopProducing(self):
         """
-        @see L{IProducer.stopProducing}
+        @see: C{IProducer.stopProducing}
```
Comment (rodrigc) [inline twisted/web/_http2.py]: No, @see: is the only good change here

File: twisted/web/_http2.py [hunk 24]
```diff
@@ -1073,22 +1073,22 @@ def unregisterProducer(self):
...
         """
         self.producing = False
         self.abortConnection()
 
 
     def pauseProducing(self):
         """
-        @see L{IPushProducer.pauseProducing}
+        @see: C{IPushProducer.pauseProducing}
```
Comment (rodrigc) [inline twisted/web/_http2.py]: No, see above

File: twisted/web/_http2.py [hunk 25]
```diff
@@ -1073,22 +1073,22 @@ def unregisterProducer(self):
...
         """
         self.producing = False
 
 
     def resumeProducing(self):
         """
-        @see L{IPushProducer.resumeProducing}
+        @see: C{IPushProducer.resumeProducing}
```
Comment (rodrigc) [inline twisted/web/_http2.py]: No, see above

File: twisted/trial/_dist/workerreporter.py [hunk 26]
```diff
@@ -106,7 +106,7 @@ def _getTodoReason(self, todo):
         """
         Get the reason for a C{Todo}.
 
-        If C[todo} is L{None}, return a sensible default.
+        If C{todo} is C{None}, return a sensible default.
```
Comment (rodrigc) [inline twisted/trial/_dist/workerreporter.py]: The only part of this line which is OK is changing C[todo} to C{todo}

File: twisted/persisted/styles.py [hunk 27]
```diff
@@ -103,15 +103,15 @@ def unpickleMethod(im_name, im_self, im_class):
...
 
     @param f: The function to reduce.
-    @type f: L{types.FunctionType}
+    @type f: C{types.FunctionType}
 
-    @return: a 2-tuple of a reference to L{_unpickleFunction} and a tuple of
+    @return: a 2-tuple of a reference to C{_unpickleFunction} and a tuple of
         its arguments, a 1-tuple of the function's fully qualified name.
-    @rtype: 2-tuple of C{(callable, native string}}
+    @rtype: 2-tuple of C{callable, native string}
```
Comment (rodrigc) [inline twisted/persisted/styles.py:114]: OK, this change is good it fixes a typo

File: twisted/persisted/styles.py [hunk 28]
```diff
@@ -103,15 +103,15 @@ def unpickleMethod(im_name, im_self, im_class):
 
 def _pickleFunction(f):
     """
-    Reduce, in the sense of L{pickle}'s C{object.__reduce__} special method, a
+    Reduce, in the sense of C{pickle}'s C{object.__reduce__} special method, a
     function object into its constituent parts.
 
     @param f: The function to reduce.
-    @type f: L{types.FunctionType}
+    @type f: C{types.FunctionType}
```
Comment (rodrigc) [inline twisted/persisted/styles.py]: No

File: twisted/persisted/styles.py [hunk 29]
```diff
@@ -103,15 +103,15 @@ def unpickleMethod(im_name, im_self, im_class):
...
 
-    @return: a 2-tuple of a reference to L{_unpickleFunction} and a tuple of
+    @return: a 2-tuple of a reference to C{_unpickleFunction} and a tuple of
```
Comment (rodrigc) [inline twisted/persisted/styles.py]: No

File: twisted/persisted/styles.py (hunk 28 above)
Comment (rodrigc) [inline twisted/persisted/styles.py]: No.

File: twisted/persisted/styles.py [hunk 30]
```diff
@@ -103,15 +103,15 @@ def unpickleMethod(im_name, im_self, im_class):
 
 def _pickleFunction(f):
     """
-    Reduce, in the sense of L{pickle}'s C{object.__reduce__} special method, a
+    Reduce, in the sense of C{pickle}'s C{object.__reduce__} special method, a
```
Comment (rodrigc) [inline twisted/persisted/styles.py]: No.

File: twisted/names/server.py [hunk 31]
```diff
@@ -70,22 +70,22 @@ class DNSServerFactory(protocol.ServerFactory):
...
         @param clients: Resolvers which are capable of performing recursive DNS
             lookups.
-        @type clients: L{list} of L{IResolver} providers
+        @type clients: C{list} of C{IResolver} providers
 
         @param verbose: An integer controlling the verbosity of logging of
             queries and responses. Default is C{0} which means no logging. Set
             to C{2} to enable logging of full query and response messages.
-        @param verbose: L{int}
+        @type verbose: C{int}
```
Comment (rodrigc) [inline twisted/names/server.py]: This line should be @type verbose: L{int}

File: twisted/names/server.py [hunk 32]
```diff
@@ -70,22 +70,22 @@ class DNSServerFactory(protocol.ServerFactory):
...
             answers. The first cache instance is assigned to
             C{DNSServerFactory.cache} and its C{cacheResult} method will be
             called when a response is received from one of C{clients}.
-        @type caches: L{list} of L{Cache<twisted.names.cache.Cache} instances
+        @type caches: C{list} of C{Cache<twisted.names.cache.Cache} instances
 
         @param clients: Resolvers which are capable of performing recursive DNS
             lookups.
-        @type clients: L{list} of L{IResolver} providers
+        @type clients: C{list} of C{IResolver} providers
```
Comment (rodrigc) [inline twisted/names/server.py]: No

File: twisted/names/server.py [hunk 33]
```diff
@@ -70,22 +70,22 @@ class DNSServerFactory(protocol.ServerFactory):
...
         @param authorities: Resolvers which provide authoritative answers.
-        @type authorities: L{list} of L{IResolver} providers
+        @type authorities: C{list} of C{IResolver} providers
 
         @param caches: Resolvers which provide cached non-authoritative
             answers. The first cache instance is assigned to
             C{DNSServerFactory.cache} and its C{cacheResult} method will be
             called when a response is received from one of C{clients}.
-        @type caches: L{list} of L{Cache<twisted.names.cache.Cache} instances
+        @type caches: C{list} of C{Cache<twisted.names.cache.Cache} instances
```
Comment (rodrigc) [inline twisted/names/server.py]: No.  However, there is a problem on this line, but you didn't fix it.  It should be:

@type caches: L{list} of L{Cache<twisted.names.cache.Cache>} instances

File: twisted/names/server.py [hunk 34]
```diff
@@ -70,22 +70,22 @@ class DNSServerFactory(protocol.ServerFactory):
     def __init__(self, authorities=None, caches=None, clients=None, verbose=0):
         """
         @param authorities: Resolvers which provide authoritative answers.
-        @type authorities: L{list} of L{IResolver} providers
+        @type authorities: C{list} of C{IResolver} providers
```
Comment (rodrigc) [inline twisted/names/server.py]: No

File: twisted/names/server.py [hunk 35]
```diff
@@ -27,38 +27,38 @@
...
         using this object as their controller.
     @type connections: C{list} of L{DNSProtocol} instances
 
     @ivar protocol: A callable used for building a DNS stream protocol. Called
-        by L{DNSServerFactory.buildProtocol} and passed the L{DNSServerFactory}
+        by C{DNSServerFactory.buildProtocol} and passed the C{DNSServerFactory}
         instance as the one and only positional argument.  Defaults to
-        L{dns.DNSProtocol}.
-    @type protocol: L{IProtocolFactory} constructor
+        C{dns.DNSProtocol}.
```
Comment (rodrigc) [inline twisted/names/server.py]: No

File: twisted/names/server.py [hunk 36]
```diff
@@ -27,38 +27,38 @@
...
+    @ivar verbose: See C{__init__}
 
-    @ivar connections: A list of all the connected L{DNSProtocol} instances
+    @ivar connections: A list of all the connected C{DNSProtocol} instances
         using this object as their controller.
     @type connections: C{list} of L{DNSProtocol} instances
 
     @ivar protocol: A callable used for building a DNS stream protocol. Called
-        by L{DNSServerFactory.buildProtocol} and passed the L{DNSServerFactory}
+        by C{DNSServerFactory.buildProtocol} and passed the C{DNSServerFactory}
```
Comment (rodrigc) [inline twisted/names/server.py]: No

File: twisted/names/server.py [hunk 37]
```diff
@@ -27,38 +27,38 @@
...
         C{authorities}, C{caches} and C{clients} to which queries will be
         dispatched.
-    @type resolver: L{resolve.ResolverChain}
+    @type resolver: C{resolve.ResolverChain}
 
-    @ivar verbose: See L{__init__}
+    @ivar verbose: See C{__init__}
 
-    @ivar connections: A list of all the connected L{DNSProtocol} instances
+    @ivar connections: A list of all the connected C{DNSProtocol} instances
```
Comment (rodrigc) [inline twisted/names/server.py]: No

File: twisted/names/server.py [hunk 38]
```diff
@@ -27,38 +27,38 @@
...
 
-    @ivar resolver: A L{resolve.ResolverChain} containing an ordered list of
+    @ivar resolver: A C{resolve.ResolverChain} containing an ordered list of
         C{authorities}, C{caches} and C{clients} to which queries will be
         dispatched.
-    @type resolver: L{resolve.ResolverChain}
+    @type resolver: C{resolve.ResolverChain}
 
-    @ivar verbose: See L{__init__}
+    @ivar verbose: See C{__init__}
```
Comment (rodrigc) [inline twisted/names/server.py]: No

File: twisted/names/server.py [hunk 39]
```diff
@@ -27,38 +27,38 @@
...
         performing recursive DNS resolution.
-    @type canRecurse: L{bool}
+    @type canRecurse: C{bool}
 
-    @ivar resolver: A L{resolve.ResolverChain} containing an ordered list of
+    @ivar resolver: A C{resolve.ResolverChain} containing an ordered list of
         C{authorities}, C{caches} and C{clients} to which queries will be
         dispatched.
-    @type resolver: L{resolve.ResolverChain}
+    @type resolver: C{resolve.ResolverChain}
```
Comment (rodrigc) [inline twisted/names/server.py]: No

File: twisted/names/server.py (hunk 19 above)
Comment (rodrigc) [inline twisted/names/server.py]: No

File: twisted/names/server.py [hunk 40]
```diff
@@ -27,38 +27,38 @@
...
     provides records for responses to DNS queries.
 
-    @ivar cache: A L{Cache<twisted.names.cache.Cache>} instance whose
+    @ivar cache: A C{Cache<twisted.names.cache.Cache>} instance whose
         C{cacheResult} method is called when a response is received from one of
         C{clients}. Defaults to L{None} if no caches are specified. See
-        C{caches} of L{__init__} for more details.
-    @type cache: L{Cache<twisted.names.cache.Cache} or L{None}
+        C{caches} of C{__init__} for more details.
+    @type cache: C{Cache<twisted.names.cache.Cache} or C{None}
```
Comment (rodrigc) [inline twisted/names/server.py]: No.  There is one problem on this line though, but you did not fix it.
This should be:
@type cache: L{Cache<twisted.names.cache.Cache>} or L{None}

File: twisted/names/server.py [hunk 41]
```diff
@@ -27,38 +27,38 @@
 
 class DNSServerFactory(protocol.ServerFactory):
     """
-    Server factory and tracker for L{DNSProtocol} connections.  This class also
+    Server factory and tracker for C{DNSProtocol} connections.  This class also
     provides records for responses to DNS queries.
 
-    @ivar cache: A L{Cache<twisted.names.cache.Cache>} instance whose
+    @ivar cache: A C{Cache<twisted.names.cache.Cache>} instance whose
```
Comment (rodrigc) [inline twisted/names/server.py]: No

File: twisted/names/server.py [hunk 42]
```diff
@@ -27,38 +27,38 @@
 
 class DNSServerFactory(protocol.ServerFactory):
     """
-    Server factory and tracker for L{DNSProtocol} connections.  This class also
+    Server factory and tracker for C{DNSProtocol} connections.  This class also
```
Comment (rodrigc) [inline twisted/names/server.py]: No

File: twisted/names/dns.py [hunk 43]
```diff
@@ -2637,23 +2637,23 @@ def toStr(self):
...
 
-        @param message: The source L{Message}.
-        @type message: L{Message}
+        @param message: The source C{Message}.
+        @type message: C{Message}
 
-        @return: A new L{_EDNSMessage}
-        @rtype: L{_EDNSMessage}
+        @return: A new C{_EDNSMessage}
+        @rtype: C{_EDNSMessage}
```
Comment (rodrigc) [inline twisted/names/dns.py]: No

File: twisted/names/dns.py [hunk 44]
```diff
@@ -2637,23 +2637,23 @@ def toStr(self):
...
+        specific attributes C{extendedRCODE}, C{ednsVersion}, C{dnssecOK},
+        C{ednsOptions}.
 
         The C{extendedRCODE} will be combined with C{message.rCode} and assigned
         to C{self.rCode}.
 
-        @param message: The source L{Message}.
-        @type message: L{Message}
+        @param message: The source C{Message}.
+        @type message: C{Message}
```
Comment (rodrigc) [inline twisted/names/dns.py]: No

File: twisted/names/dns.py [hunk 45]
```diff
@@ -2637,23 +2637,23 @@ def toStr(self):
...
+        Construct and return a new C{_EDNSMessage} whose attributes and records
+        are derived from the attributes and records of C{message} (a C{Message}
+        instance).
 
-        If present, an I{OPT} record will be extracted from the C{additional}
+        If present, an C{OPT} record will be extracted from the C{additional}
         section and its attributes and options will be used to set the EDNS
-        specific attributes C{extendedRCODE}, c{ednsVersion}, c{dnssecOK},
-        c{ednsOptions}.
+        specific attributes C{extendedRCODE}, C{ednsVersion}, C{dnssecOK},
```
Comment (rodrigc) [inline twisted/names/dns.py:2646]: OK this fix is good

File: twisted/names/dns.py [hunk 46]
```diff
@@ -2637,23 +2637,23 @@ def toStr(self):
...
         """
-        Construct and return a new L(_EDNSMessage} whose attributes and records
-        are derived from the attributes and records of C{message} (a L{Message}
-        instance)
+        Construct and return a new C{_EDNSMessage} whose attributes and records
+        are derived from the attributes and records of C{message} (a C{Message}
+        instance).
 
-        If present, an I{OPT} record will be extracted from the C{additional}
+        If present, an C{OPT} record will be extracted from the C{additional}
```
Comment (rodrigc) [inline twisted/names/dns.py:2644]: OK, this fix is good

File: twisted/names/dns.py [hunk 47]
```diff
@@ -2637,23 +2637,23 @@ def toStr(self):
     @classmethod
     def _fromMessage(cls, message):
         """
-        Construct and return a new L(_EDNSMessage} whose attributes and records
```
Comment (rodrigc) [inline twisted/names/dns.py:2640]: No, change this bad.  There is one bug on this line but you didn't fix it.
This should be:
L{_EDNSMessage}

File: twisted/names/dns.py [hunk 48]
```diff
@@ -2455,12 +2455,12 @@ def __init__(self, id=0, answer=False, opCode=OP_QUERY, auth=False,
                  checkingDisabled=False, maxSize=512,
                  queries=None, answers=None, authority=None, additional=None):
         """
-        Construct a new L{_EDNSMessage}
+        Construct a new C{_EDNSMessage}
```
Comment (rodrigc) [inline twisted/names/dns.py]: No

File: twisted/names/dns.py [hunk 49]
```diff
@@ -772,10 +772,10 @@ class _OPTVariableOption(tputil.FancyStrMixin, tputil.FancyEqMixin, object):
     """
     A class to represent OPT record variable options.
 
-    @see: L{_OPTVariableOption.__init__} for documentation of public
+    @see: C{_OPTVariableOption.__init__} for documentation of public
         instance attributes.
 
-    @see: L{https://tools.ietf.org/html/rfc6891#section-6.1.2}
+    @see: C{https://tools.ietf.org/html/rfc6891#section-6.1.2}
```
Comment (rodrigc) [inline twisted/names/dns.py]: No, this should be U{

File: twisted/names/dns.py [hunk 50]
```diff
@@ -772,10 +772,10 @@ class _OPTVariableOption(tputil.FancyStrMixin, tputil.FancyEqMixin, object):
     """
     A class to represent OPT record variable options.
 
-    @see: L{_OPTVariableOption.__init__} for documentation of public
+    @see: C{_OPTVariableOption.__init__} for documentation of public
```
Comment (rodrigc) [inline twisted/names/dns.py]: No

File: twisted/names/dns.py [hunk 51]
```diff
@@ -601,13 +601,13 @@ class _OPTHeader(tputil.FancyStrMixin, tputil.FancyEqMixin, object):
...
-        (C{dns.OPT} for OPT Record. This attribute is a readonly
+        C{dns.OPT} for OPT Record. This attribute is a readonly
         property.
 
-    @see: L{_OPTHeader.__init__} for documentation of other public
+    @see: C{_OPTHeader.__init__} for documentation of other public
         instance attributes.
 
-    @see: L{https://tools.ietf.org/html/rfc6891#section-6.1.2}
+    @see: C{https://tools.ietf.org/html/rfc6891#section-6.1.2}
```
Comment (rodrigc) [inline twisted/names/dns.py]: No, this should be U{

File: twisted/names/dns.py [hunk 52]
```diff
@@ -601,13 +601,13 @@ class _OPTHeader(tputil.FancyStrMixin, tputil.FancyEqMixin, object):
         attribute is a readonly property.
 
     @ivar type: The DNS record type. This is a fixed value of 41
-        (C{dns.OPT} for OPT Record. This attribute is a readonly
+        C{dns.OPT} for OPT Record. This attribute is a readonly
         property.
 
-    @see: L{_OPTHeader.__init__} for documentation of other public
+    @see: C{_OPTHeader.__init__} for documentation of other public
```
Comment (rodrigc) [inline twisted/names/dns.py]: No

File: twisted/internet/interfaces.py [hunk 53]
```diff
@@ -1007,20 +1007,20 @@ def adoptDatagramPort(fileDescriptor, addressFamily, protocol,
...
         @type fileDescriptor: C{int}
 
-        @param addressFamily: The address family (or I{domain}) of the socket.
-            For example, L{socket.AF_INET6}.
+        @param addressFamily: The address family or I{domain} of the socket.
+            For example, C{socket.AF_INET6}.
         @type addressFamily: C{int}
 
-        @param protocol: A L{DatagramProtocol} instance to connect to
+        @param protocol: A C{DatagramProtocol} instance to connect to
```
Comment (rodrigc) [inline twisted/internet/interfaces.py]: No

File: twisted/internet/interfaces.py [hunk 54]
```diff
@@ -1007,20 +1007,20 @@ def adoptDatagramPort(fileDescriptor, addressFamily, protocol,
             done as soon as C{adoptDatagramPort} returns.
         @type fileDescriptor: C{int}
 
-        @param addressFamily: The address family (or I{domain}) of the socket.
-            For example, L{socket.AF_INET6}.
+        @param addressFamily: The address family or I{domain} of the socket.
```
Comment (rodrigc) [inline twisted/internet/interfaces.py:1010]: This part is OK, keep this

File: twisted/python/logfile.py (hunk 20 above)
Comment (acabhishek942) [inline twisted/python/logfile.py:65]: http://epydoc.sourceforge.net/epytext.html . Please see section 3.1 for unordered list

Comment (rodrigc) [inline twisted/python/logfile.py:65]: Thanks for the explanation.
This is OK.

**Comment by rodrigc** [comment]
Merged in https://github.com/twisted/twisted/pull/217
Thanks for the submission!
